math_functions.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+def calculate_triangular_number(num: int) -> int:
+    if num <= 0:
+        return 0
+    return sum(range(num + 1))

```

Comment:
[Original comment](https://github.com/nadavtzaysler/CleanCode/pull/1#discussion_r1628416113)

Although the name sounds great, it's not very accurate as not all positive integers belong to the triangular numbers sequence (1, 3, 6, 10, 15...).
I more suitable name would be `calculate_sum_up_to` or something like that.

Reply:
Also, [here](https://github.com/nadavtzaysler/CleanCode/pull/1/files#r1628464183).